README.md

```diff
@@ -1,14 +1,49 @@
 # Experimental Synthetics Agent
 
 Synthetic Monitoring with Real Browsers.
-**Please note this is an unsupported experimental project. Expect things to break and change!** 
+**Please note this is an unsupported experimental project. Expect things to break and change!**
 
-Docs are forthcoming, feel free to browse the repository, but we ask that you refrain from asking questions about use till docs
-are published.
+## Usage
 
-If you'd like to stay informed about synthetic monitoring development please visit [our synthetic monitoring page](https://www.elastic.co/what-is/synthetic-monitoring) where you can sign up to be notified of our initial release.
+Install the package
 
-## CI
+```sh
+npm install -g @elastic/synthetics
+```
 
-The CI will not run builds for collaborators PRs that are not approval by a members of Elastic org,
-Elastic Users can launch the CI on PRs by putting a comment like `/test` or manually on the CI.
\ No newline at end of file
+Run the suites via CLI
+
+```
+npx synthetics [options] [dir] [files] file
+
+Options:
+  -s, --suite-params <jsonstring>  Variables (default: "{}")
+  -e, --environment <envname>      e.g. production (default: "development")
+  -j, --json                       output newline delimited JSON
+  -d, --debug                      print debug logs info
+  --pattern <pattern>              RegExp file patterns to search inside directory
+  --inline                         Run inline journeys from heartbeat
+  -r, --require <modules...>       module(s) to preload
+  --no-headless                    run browser in headful mode
+  --screenshots                    take screenshots between steps (only shown in some reporters)
+  --network                        capture all network information for all steps
+  --metrics                        capture performance metrics for each step
+  --dry-run                        don't actually execute anything, report only registered journeys
+  --journey-name <name>            only run the journey with the given name
+  --outfd <fd>                     specify a file descriptor for logs. Default is stdout
+  -V, --version                    output the version number
+  -h, --help                       display help for command
+```
+
+## Documentation
+
+Documentation is avaiable on [elastic.co](https://www.elastic.co/guide/en/observability/7.10/synthetic-monitoring.html)
```

Comment:
Now that 7.10 docs are live, can you change this to `current` so we don't have to update every release?
```suggestion
Documentation is avaiable on [elastic.co](https://www.elastic.co/guide/en/observability/current/synthetic-monitoring.html)
```